impeller/compiler/switches.cc

```diff
@@ -196,7 +196,9 @@ bool Switches::AreValid(std::ostream& explain) const {
   }
 
   if (!working_directory || !working_directory->is_valid()) {
-    explain << "Could not figure out working directory." << std::endl;
+    explain << "Could not open the working directory: "
```

Comment:
Consider wrapping in quotes to make it clear if/when the string is empty.

Reply:
Done.